src/Exception/ConfigurationException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cli for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cli/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cli/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Laminas\Cli\Exception;
+
+use RuntimeException;
+
+use function sprintf;
+
+final class ConfigurationException extends RuntimeException implements ExceptionInterface
+{
+    public static function withInvalidMappedCommandClass(string $commandName, string $targetClassName): self
+    {
+        return new static(sprintf(
```

Comment:
```suggestion
        return new self(sprintf(
```

Since the class is `final`, we can use `self` safely here.